src/core/api/email_test.go

```diff
@@ -12,6 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Avoid to run this test in local env (Mac)
```

Comment:
You can add parm to `go test` when you run the test cases locally?

Reply:
I run it in GoLand